frontend_tests/node_tests/popovers.js

```diff
@@ -31,6 +31,7 @@ const message_lists = mock_esm("../../static/js/message_lists", {
 });
```

Comment:
<img width="445" alt="Screenshot 2022-11-18 at 7 54 09 PM" src="https://user-images.githubusercontent.com/25124304/202726796-99cfa34f-abef-44ad-b47e-37a4c1ca9f0f.png">

Reply:
Notice that username is trimmed too. I am not sure if there is any other solution to that.